CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+If you don't have git on your machine, [install it](https://help.github.com/articles/set-up-git/).
+
+## The beginner's guide to contributing to a GitHub project
+
+<img align="right" width="300" src="assets/images/fork.png" alt="fork this repository" />
+
+## Fork this repository
+
+Fork this repository by clicking on the fork button on the top of this page.
+This will create a copy of this repository in your account.
+
+## Clone the repository
+
+<img align="right" width="300" src="assets/images/clone.png" alt="clone this repository" />
+
+Now clone the forked repository to your machine. Go to your GitHub account, open the forked repository, click on the code button and then click the _copy to clipboard_ icon.
+
+Open a terminal and run the following git command:
+
+```
+git clone "url you just copied"
+```
+
+where "url you just copied" (without the quotation marks) is the url to this repository (your fork of this project). See the previous steps to obtain the url.
```

Comment:
Let's make it more obvious 😉 
you can put the actual repo url here. Easier for people to copy 😄 

And then you could remove the line which explains what the "url you just copied" is 🙂

Reply:
yeah, that makes sense 😄